frontend/src/metabase-types/analytics/search.ts

```diff
@@ -44,10 +44,19 @@ type SnowplowSearchContext =
 // TODO: once we've finalized the new context names, we should add them to the Snowplow schema.
 // It would also be good to use an exclusion list, so that this is easier to maintain when new context are added.
 const RegisteredSearchContext = [
-  "search-app",
-  "search-bar",
+  "available-models",
   "command-palette",
   "entity-picker",
+  "metabot",
+  "metrics-browser",
+  "model-upload",
+  "model-browser",
+  "question-picker",
+  "search-app",
+  "search-bar",
+  "skip-token", // never sent to the backend, used in a sentinel payload
```

Comment:
I don't think this value can ever make it this far. If it does, that should be an error.